README.md

```diff
@@ -1,2 +1,276 @@
+   - devicetree source files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+# Getting Started
+
+## Creating a New Module
+
+This module is designed to be a boilerplate for other modules simply by:
+
+1. cloning
+
+1. stripping out any unnecessary content (e.g. no drivers or subsystems)
+
+  - If you don't have any non-sample, non-test projects:
+
+    1. remove the `zephyr/projects/` subdirectory
+    1. In `zephyr/modules.yml` remove the reference to `zephyr/projects`
+
+
+## Creating a Free-Standing Manifest Repository
+
+This example module is also an appropriate template for creating a
+manifest-only repository because, like modules that also contain a
+manifest file, they may contain the following:
+
+- documentation describing the scope and purpose of the workspace
+  configured by the manifest
+
+- configuration settings for CI pipelines
+
+- scripts unique to the breadth of the workspace
+
+To strip this repository down to be a manifest-only repository:
+
+  1. Remove all files and subdirectories referenced by `zephyr/module.yml`
+  1. Remove the `zephyr/include` subdirectory
+  1. Remove `zephyr/module.yml`
+
+
+## Checking Out Code
+
+The Zephyr module ZXA may be cloned and added to the Zephyr build system
+in exactly the same way as any other repository.
+
+Once the Zephyr development environment has been set up (see [Zephyr Getting Started Guide][3]), then follow instructions here to:
+
+- Get the source code with Zephyr module ZXA as the `west`
+  manifest repository
+- Build and run projects
+
+
+## Get Zephyr and Install Python Dependencies
+
+Clone Zephyr module ZXA and its dependencies into a new `west`
+workspace named :file:`zp`. You'll also install additional Python
+dependencies.
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        ~/zxa
+      cd ~/zxa
+      west update
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+
+## Build and Run the zxa_example_project
+
+Build the `zxa_example_project` with
+`west build`, changing ``<your-board-name>``
+appropriately for your board:
+
+### Ubuntu
+
+1. Build projects directly from the command line
+
+      ```bash
+      cd ~/zxa
+      west build -p auto -b native_posix \
+        example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bash
+      ./build/example-application/zephyr/projects/zxa_example_project/native_posix/zephyr/zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
```

Comment:
Do we really need to introduce twister at this stage? Can we please have a simple GSG and then have a "Next steps" or "Advanced  topics" doc page that can deal with this?

Reply:
Yes. Unless we provide multiple example repositories, then this one must illustrate all the functionality supported through `module.yml`. 

It is of very little benefit to only document what can be done.  This is already described in the Zephyr documentation.  The benefit of the example repository is to have a maintained, continuously verified skeletal **implementation** of a fully connected module.